content/blog/tidyr-1-3-0/index.Rmd

```diff
@@ -0,0 +1,289 @@
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("tidyr")
+```
+
+This post highlights the biggest changes in this release:
+
+-   A new family of `separate_*()` functions supersede `separate()` and `extract()` and come with useful debugging features.
+
+-   `unnest_wider()` and `unnest_longer()` gain a bundle of useful improvements.
+
+-   `pivot_longer()` gets a new `cols_vary` argument.
+
+-   `nest(.by)` provides a new (and hopefully final) way to create nested datasets.
+
+You should also notice generally improved errors with this release: we check function arguments more aggressively and take care to always report the name of the function that you called, not some internal helper.
+As usual, you can find a full set of changes in the [release notes](http://github.com/tidyverse/tidyr/releases/tag/v1.3.0).
+
+```{r setup}
+library(tidyr)
+library(dplyr, warn.conflicts = FALSE)
+```
+
+## `separate_*()` family of functions
+
+The biggest feature of this release is a new, experimental, family of functions for separating string columns:
+
+|                                  | Make columns                | Make rows                    |
+|--------------------------|-----------------------|-----------------------|
+| Separate with delimiter          | `separate_wider_delim()`    | `separate_longer_delim()`    |
+| Separate by position             | `separate_wider_position()` | `separate_longer_position()` |
+| Separate with regular expression | `separate_wider_regex()`    |                              |
+
+These functions collectively supersede `extract()`, `separate()`, and `separate_rows()` because they have more consistent names and arguments, have better performance (thanks to stringr), and provide a new approach for handling problems.
+
+|                                  | Make columns                     | Make rows         |
+|---------------------------|---------------------------|------------------|
+| Separate with delimiter          | `separate(sep = string)`         | `separate_rows()` |
+| Separate by position             | `separate(sep = integer vector)` | N/A               |
+| Separate with regular expression | `extract()`                      |                   |
+
+Here I'll focus on the `wider` functions because they generally present the most interesting challenges.
+Let's start by grabbing some census data with the [tidycensus](https://walker-data.com/tidycensus/) package:
+
+```{r}
+vt_census <- tidycensus::get_decennial(
+  geography = "block",
+  state = "VT",
+  county = "Washington",
+  variables = "P1_001N",
+  year = 2020
+)
+vt_census
+```
+
+The `GEOID` column is made up of four components: a 2 digit state identifier, a 3 digit county identifier, a 6 digit tract identifier, and a 4 digit block identifier.
+We could use `separate_wider_position()` to extract these into their own variables:
```

Comment:
Is there a reason you have "could" over "can" here?

Reply:
no?